README.md

```diff
@@ -52,34 +90,25 @@ use {'nvim-orgmode/orgmode',
     }
 ```
 
-- [vim-plug](https://github.com/junegunn/vim-plug)
-
-```vim
-Plug 'nvim-treesitter/nvim-treesitter'
-Plug 'nvim-orgmode/orgmode'
-```
-
-- [dein](https://github.com/Shougo/dein.vim)
-
-```vim
-call dein#add('nvim-treesitter/nvim-treesitter')
-call dein#add('nvim-orgmode/orgmode')
-```
+</details>
 
-## Setup
+### Setup
 
 ```lua
 -- init.lua
 
--- Load custom tree-sitter grammar for org filetype
+-- Load custom treesitter grammar for org filetype
```

Comment:
I chose to follow the wording of nvim-treesitter and not that from the original project called 'Tree-sitter'